### DISCUSSION TITLE
[Bug] bg-gradient is not work and vscode does not prompt

### DISCUSSION THREAD

**Discussion by saddit** [tailwindlabs/tailwindcss#6562] [Help]
**What version of Tailwind CSS are you using?**

3.0.5

**What build tool (or framework if it abstracts the build tool) are you using?**

- vite 2.6.4 
- vue 3.2.16

**What version of Node.js are you using?**

newest

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Describe your issue**

bg-gradient is not work and vscode does not prompt

**Comment by gaedlld** [comment]
Same issue for me, on Mac OS Safari with React !
